stix_shifter_modules/elastic_ecs/stix_translation/json/to_stix_map.json

```diff
@@ -45,7 +45,93 @@
         "object": "src_ip",
         "references": ["src_mac"]
       }
-    ]
+    ],
+    "bytes": {
+      "key": "network-traffic.src_byte_count",
+      "object": "nt"
+    },
+    "packets": {
+      "key": "network-traffic.src_packets",
+      "object": "nt"
+    },
+    "as": {
+      "number": [
+        {
+          "key": "autonomous-system.number",
+          "object": "autonomous-system"
+        },
+        {
+          "key": "ipv4-addr.belongs_to_refs",
+          "object": "src_ip",
+          "references": ["autonomous-system"]
+        }
+      ],
+      "organization": {
+        "name" : {
+          "key": "autonomous-system.name"
+        }
+      }
+    },
+    "address": {
+      "key": "x_ecs_source.address",
```

Comment:
Now that we are putting the custom properties as custom cybox objects, the naming convention will have to change from underscores to dashes, so in this case it would be `x-ecs-source.address`

Reply:
updated